app/assets/javascripts/disable-button-after-submit.js

```diff
@@ -0,0 +1,4 @@
+$('form').submit(function (event) {
+  $(this).children('input[type=submit]').prop('disabled', true)
+  $(this).children('input[type=submit]').prop('value', 'Please wait...')
```

Comment:
remove, should not 'flash' messages to user which will confuse in the normal case of minimal delay between clicking and page updates.